Doc/library/sqlite3.rst

```diff
@@ -1230,60 +1245,15 @@ The exception hierarchy is defined by the DB-API 2.0 (:pep:`249`).
    ``NotSupportedError`` is a subclass of :exc:`DatabaseError`.
 
 
-.. _sqlite3-types:
+.. _sqlite3-guides:
 
-SQLite and Python types
------------------------
-
-
-Introduction
-^^^^^^^^^^^^
-
-SQLite natively supports the following types: ``NULL``, ``INTEGER``,
-``REAL``, ``TEXT``, ``BLOB``.
-
-The following Python types can thus be sent to SQLite without any problem:
-
-+-------------------------------+-------------+
-| Python type                   | SQLite type |
-+===============================+=============+
-| :const:`None`                 | ``NULL``    |
-+-------------------------------+-------------+
-| :class:`int`                  | ``INTEGER`` |
-+-------------------------------+-------------+
-| :class:`float`                | ``REAL``    |
-+-------------------------------+-------------+
-| :class:`str`                  | ``TEXT``    |
-+-------------------------------+-------------+
-| :class:`bytes`                | ``BLOB``    |
-+-------------------------------+-------------+
-
-
-This is how SQLite types are converted to Python types by default:
-
-+-------------+----------------------------------------------+
-| SQLite type | Python type                                  |
-+=============+==============================================+
-| ``NULL``    | :const:`None`                                |
-+-------------+----------------------------------------------+
-| ``INTEGER`` | :class:`int`                                 |
-+-------------+----------------------------------------------+
-| ``REAL``    | :class:`float`                               |
-+-------------+----------------------------------------------+
-| ``TEXT``    | depends on :attr:`~Connection.text_factory`, |
-|             | :class:`str` by default                      |
-+-------------+----------------------------------------------+
-| ``BLOB``    | :class:`bytes`                               |
-+-------------+----------------------------------------------+
-
-The type system of the :mod:`sqlite3` module is extensible in two ways: you can
-store additional Python types in an SQLite database via object adaptation, and
-you can let the :mod:`sqlite3` module convert SQLite types to different Python
-types via converters.
+Guides
+------
 
+.. _sqlite3-adapters:
 
 Using adapters to store custom Python types in SQLite databases
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
FWIW https://devguide.python.org/documenting/#sections recommends `^^^` after `---`.  I'm not sure how consistent its use is in the docs, and it doesn't matter too much.

Reply:
Right. Where does `~` belong?

Reply:
It isn't one of the six heading levels defined there (and going deeper than that is likely ill advised)—not the most important, but it does make it easier to move content between documents and keep track of what heading level you are at in the source.